keytree/src/lib.rs

```diff
@@ -174,25 +173,18 @@ mod tests {
         let mut rng = ChaChaRng::from_seed(seed);
         let xprv = Xprv::random(&mut rng);
         let xprv_bytes = xprv.to_bytes();
+        let xprv_hex = hex::encode(&xprv_bytes[..]);
 
-        // hardcoded, but happens to be expected_scalar concatenated with expected_dk
-        let expected_bytes = [
-            74, 83, 195, 251, 188, 89, 151, 14, 229, 248, 90, 248, 19, 135, 93, 255, 193, 58, 144,
-            74, 46, 83, 174, 126, 101, 250, 13, 234, 110, 98, 201, 1, 159, 7, 231, 190, 85, 81, 56,
-            122, 152, 186, 151, 124, 115, 45, 8, 13, 203, 15, 41, 160, 72, 227, 101, 105, 18, 198,
-            83, 62, 50, 238, 122, 237,
-        ];
-        assert_eq!(&xprv_bytes[..], &expected_bytes[..]);
+        assert_eq!(
+            xprv_hex,
+            "4a53c3fbbc59970ee5f85af813875dffc13a904a2e53ae7e65fa0dea6e62c9019f07e7be5551387a98ba977c732d080dcb0f29a048e3656912c6533e32ee7aed"
+        );
     }
 
     #[test]
     fn deserialize_xprv_test() {
-        let xprv_bytes = [
-            74, 83, 195, 251, 188, 89, 151, 14, 229, 248, 90, 248, 19, 135, 93, 255, 193, 58, 144,
-            74, 46, 83, 174, 126, 101, 250, 13, 234, 110, 98, 201, 1, 159, 7, 231, 190, 85, 81, 56,
-            122, 152, 186, 151, 124, 115, 45, 8, 13, 203, 15, 41, 160, 72, 227, 101, 105, 18, 198,
-            83, 62, 50, 238, 122, 237,
-        ];
+        let xprv_hex =  "4a53c3fbbc59970ee5f85af813875dffc13a904a2e53ae7e65fa0dea6e62c9019f07e7be5551387a98ba977c732d080dcb0f29a048e3656912c6533e32ee7aed";
+        let xprv_bytes = hex::decode(xprv_hex).unwrap();
```

Comment:
@oleganza I know you said there shouldn't be any decoding in these tests, but I think it's unavoidable here, given that `from_bytes` expects, well, a byte array. 

(I'll also update this to inline `xprv_hex`.)

Reply:
A hypothetical `to_hex64` helper can take `[u8;64]`. But `&to_bytes()[..]` is not too bad imho.